pkgs/by-name/br/bruno/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  dpkg,
+  makeWrapper,
+  wrapGAppsHook,
+  alsa-lib,
+  gtk3,
+  mesa,
+  nspr,
+  nss,
+  systemd,
+  ...
+}:
+stdenv.mkDerivation rec {
+  pname = "bruno";
+  version = "0.16.5";
```

Comment:
0.17 is available right now

https://github.com/usebruno/bruno/releases/tag/v0.17.0